### ISSUE TITLE
Label on arrows

### ISSUE THREAD

**Issue by Abhay2012** [adrai/flowchart.js#134]
i want add some text on arrows, is there any function or property by which i can achieve this?

**Comment by adrai** [comment]
Currently not possible.

**Comment by Abhay2012** [comment]
okay, is it possible to create arrow on left side of decision box, right now we can create arrows on right side and bottom, i want to create arrow on right and left side, is it possible??

**Comment by adrai** [comment]
No, currently this is the default algorithm... (https://github.com/adrai/flowchart.js/issues/15#issuecomment-37029886)

**Comment by Abhay2012** [comment]
okay, thanks

**Comment by mohamedganiwbc** [comment]
Unable set Dynamic Options in chart.drawSVG()
